gossip/src/cluster_info.rs

```diff
@@ -2794,6 +2795,8 @@ pub struct NodeConfig {
     pub public_tpu_forwards_addr: Option<SocketAddr>,
 }
 
+const QUIC_ENDPOINTS: usize = 10;
```

Comment:
we're going with `10`?

Reply:
I tried with 4 -- there was about 20% degradation in term of throughput. I think 10 is an okay choice for now. I have not seen much memory usage increase under the spamming tool.